js/app.js

```diff
@@ -1 +1,116 @@
-console.log('Hello frontend');
+document.addEventListener("DOMContentLoaded", function(){
+
```

Comment:
There are no comments to provide context, which normally would be a bad thing. In your case your code is fine as it is so intuitively written that it can be intuitively consumed by its reader. Really clean/awesome JS, Hailey!